backend/src/services/tests/muralcollection.test.ts

```diff
@@ -57,12 +58,13 @@ test("show valid mural collection", async () => {
   const collectionService: MuralCollectionService = new MuralCollectionService();
 
   const murals: number[] = [1, 2];
-  const create = await collectionService.create(params, murals);
-  if (!create.success) {
-    fail("Creating mural collection failed.");
-  }
+  const create = await collectionService
+    .create(params, murals)
+    .catch((err: Error) => fail());
```

Comment:
a bit late but should these `fail`s have messages like before?

Reply:
Yeah it's better if they do - there are so many though I'm kind of ignoring them rn